app/modules/auth/auth_router.py

```
@@ -270,13 +262,11 @@ async def signup(
                     )
-                    return Response(
-                        content=json.dumps(
-                            {
-                                "error": error_message,
-                                "details": f"GitHub account {provider_uid} is already linked to another user. Database constraint violation: {str(e)}",
-                            }
-                        ),
+                    return JSONResponse(
+                        content={
+                            "error": error_message,
+                            "details": f"GitHub account {provider_uid} is already linked to another user. Database constraint violation: {str(e)}",
```

Ruff (0.15.7)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)